src/stack.js

```diff
@@ -1,5 +1,49 @@
-'use strict'
```

Comment:
wow...upon further inspection you are right...I can't believe the internet lied...mind blown 😨 🔫